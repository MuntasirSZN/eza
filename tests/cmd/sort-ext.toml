bin.name = "eza"
args = "tests/itest --sort=extension"
