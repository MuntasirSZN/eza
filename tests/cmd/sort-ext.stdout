01.city  02.apple  ab  abc  test  abc.mp3  a.txt